tests/format/js/template-literals/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -231,6 +231,168 @@ throw new Error(
+          2. Go to "\${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access"
+          )}" \${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access"
+          )}
+  \`,
+\`
+# blabla \${a} \${chalk.green.underline("https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer")}
+
+    2. Go to "\${chalk.green.underline(
+      "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
+    )}"
+
+# blabla \${a} \${chalk.green.underline("https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer")}
+\`,
+]
+
+=====================================output=====================================
+[
+  \`
+      1. Go to "-{chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access"
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+  \`,
+  \`
+      2. Go to "\${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+  \`,
+  \`
+      1. Go to "-{chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access"
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+      2. Go to "\${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+  \`,
+  \`
+      2. Go to "\${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+      1. Go to "-{chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access"
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+  \`,
+  \`
+          1. Go to "-{chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access"
+          )}" \${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access",
+          )}
+      2. Go to "\${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+  \`,
+  \`
+      1. Go to "-{chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access"
+      )}" \${chalk.green.underline(
+        "https://www.npmjs.com/package/prettier/access",
+      )}
+          2. Go to "\${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access",
+          )}" \${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access",
+          )}
+  \`,
+  \`
+# blabla \${a} \${chalk.green.underline(
+    "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
+  )}
```

Comment:
This indentation looks strange, but it's a separate concern.

Reply:
I agree, the array indention should not apply to the expression.

Reply:
[One leading space makes difference, ](https://deploy-preview-13621--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEw3ADpQNpYAZYDEABAEYA2AhhZcQCTCUC+9wYAFpeQNYB0A5gCc4CXgFcoAEziDyASyhwAFBhDsYMAA4BnJAHo9Ad2O8omgLYArbb0jm9mymG6V+cB8I1yZegI5jKSUExTThJLgBpSih+SkEICAAVQWjtADMIQXMZVQBKJnwsAF0AbiwmCoKoLHRMHEKoYhIaGlZmVg4uPiERKHEpGXlFFTUNHX0jEzMrGzsHJxc3DzgvH39A4NDw8iiYuITk1IysnJB8htLyypAAGhAITRg5aG1kUDj4wwAFOIRXlC4hkoAE9XndSClnCsAMqUbIAGQUcGQaS42jg4Mh3BhjjACn4yBgwQxIDg5lIYWkknh0X4ATcADFMuZKF4YsgQJQxDAILdRuZyAB1dhyeDaXFwaF-UVyABuouBHLA2jBIAU6MEMC+KX4LJRaJJ1gAHtD8eQ4ABFMQQeD68jou6OQQajmkahwch8zSCBQwQVySQwdjIAAcAAZHfF0YKUpoOd64BrZci7v4bXBtQ9-pztABaRRhMJ84T+OTCbWuPVIVH2kno8xyQnEu7aM2W6226sGu4waj+wPBpAAJh7KTkQ34AGEIOYq5zyJ67mJ0YlqP8aw6QLKxHAAJIDWDQsA+x4AQSk0JgwPNdvRFSAA), must caused by [this `LF` check](https://github.com/prettier/prettier/blob/6d6e5177dbb9746e55706f4bece8666df578aceb/src/language-js/print/template-literal.js#L93)